### ISSUE TITLE
[BUG] Unable to load fasttext attention model

### ISSUE THREAD

**Issue by Tangolin** [GRAAL-Research/deepparse#141]
**Describe the bug**
When a `fasttext_attention` retrained model is loaded, the `handle_model_name()` function in `tools.py` incorrectly creates the model type as 'fasttext_' instead of 'fasttext', therefore we are unable to load a retrained Fasttext model.

**To Reproduce**
```
address_parser = AddressParser(
    model_type='fasttext', attention_mechanism=True, path_to_retrained_model="./path/to/fasttext_attention_model")
```

**Expected behavior**
The model type should be created correctly and the model can then be loaded.

**Screenshots**
Screenshot of the error message:
<img width="988" alt="Screenshot 2022-07-27 at 10 46 10 AM" src="https://user-images.githubusercontent.com/65639845/181149753-72899272-8fed-4ef1-be63-b166446dd3ce.png">

The part of code causing the problem:
<img width="800" alt="Screenshot 2022-07-27 at 10 48 55 AM" src="https://user-images.githubusercontent.com/65639845/181150151-62224ab2-efd0-4295-965c-d44981bd4d19.png">

**Desktop (please complete the following information):**
 - OS: Debian
 - Version 10

**Comment by davebulaval** [comment]
Hi @Tangolin,

TKS for reporting this error. It is a bug due to a change in 0.7.5. In this release, we have introduced the user named retrain parsing model, and to do so, we had to change the model naming (it was `"FasttextAttention"` before). Thus, we were not removing the underscore. Nice catch!

I have just pushed a bug-fix; as soon as tests pass, I will merge it in dev and release a bug-fix today.
